scripts/typedoc/shopify-dev-renderer/shared/index.ts

```diff
@@ -313,6 +313,32 @@ function propType(
       return `${value.value}`;
     case 'BooleanLiteralType':
       return `${value.value}`;
+    case 'TemplateLiteralType':
+      // eslint-disable-next-line no-case-declarations
+      const sortedValues = [...value.quasis, ...value.expressions].sort(
+        (first, second) => {
+          if (first.start > second.start) {
+            return 1;
+          } else if (second.start > first.start) {
+            return -1;
+          }
+          return 0;
+        },
+      );
+
+      return `&#96${sortedValues
```

Comment:
How about creating a const for `&#96` to improve readability? I think ideally the case should end with something like
```
return `${BACKTICK}${result}${BACKTICK}`;
```